src/win/tty.c

```diff
@@ -1176,13 +1185,35 @@ static int uv_tty_set_style(uv_tty_t* handle, DWORD* error) {
   char fg_bright = -1, bg_bright = -1;
   char inverse = -1;
 
+  char default_fg_color = 7, default_bg_color = 0;
+  char default_fg_bright = 0, default_bg_bright = 0;
+  char default_inverse = 0;
+
+  if (uv_tty_default_text_attributes != 0) {
```

Comment:
I don't see why you would parse it every time. Can you put all this info in a static variable, and initialize them once in `uv_tty_init()` (or, even better, a new function called `uv__tty_capture_initial_style`, and call it from `uv_tty_init()`)